providers/class-two-factor-fido-u2f.php

```diff
@@ -131,12 +120,27 @@ public function get_label() {
 	}
 
 	/**
-	 * Enqueue assets for login form.
+	 * Register script dependencies used during login and when
+	 * registering keys in the WP admin.
 	 *
-	 * @since 0.1-dev
+	 * @return void
 	 */
-	public function login_enqueue_assets() {
-		wp_enqueue_script( 'fido-u2f-login' );
+	public static function enqueue_scripts() {
+		wp_register_script(
+			'fido-u2f-api',
+			plugins_url( 'includes/Google/u2f-api.js', dirname( __FILE__ ) ),
+			null,
+			self::asset_version(),
+			true
+		);
+
+		wp_register_script(
+			'fido-u2f-login',
```

Comment:
This is used during our custom "login" page https://github.com/wordpress/two-factor/blob/a97c32b6639a3557fd149725b973615246549ffe/class-two-factor-core.php#L581-L726